package/android/src/main/java/com/mrousavy/camera/CameraView.kt

```diff
@@ -79,6 +82,7 @@ class CameraView(context: Context) : FrameLayout(context) {
   internal val cameraSession: CameraSession
   private var previewView: View? = null
   private var previewSurface: Surface? = null
+  private var previewSize: Size? = null
```

Comment:
This should be removed, the single source of truth should be `previewView?.targetSize` - that way we can't go out of sync :)